versioned_docs/version-1.0/cadence-migration-guide/index.md

```diff
@@ -213,6 +213,7 @@ Valid Input: Existing contract name in flow.json.
 $ flow migrate list-staged --network --network=testnet
 Arguments: network_name
 ```
+You can view all currently staged contracts and see reports on migration status through the [Migrations Data Repository](https://github.com/onflow/cadence/tree/master/migrations_data).
```

Comment:
The reports do not contain the staged contracts. Wasn't the plan to link to Deniz' account explorer?
https://github.com/onflow/cadence/issues/3264#issuecomment-2099480463

Reply:
```suggestion
You can view all currently staged contracts for Testnet by inspecting the account of the staging contract, `0x2ceae959ed1a7e7a`, for example through https://f.dnz.dev/0x2ceae959ed1a7e7a.

You can see reports on migration status in the [Cadence Repository](https://github.com/onflow/cadence/tree/master/migrations_data).
```